core/src/main/java/com/novoda/noplayer/mediaplayer/AndroidMediaPlayerTrackSelector.java

```diff
@@ -0,0 +1,36 @@
+package com.novoda.noplayer.mediaplayer;
+
+import android.media.MediaPlayer;
+
+import com.novoda.noplayer.PlayerAudioTrack;
+
+import java.util.ArrayList;
+
+class AndroidMediaPlayerTrackSelector {
+
+    ArrayList<PlayerAudioTrack> getAudioTracks(MediaPlayer mediaPlayer) {
+        if (mediaPlayer == null) {
+            throw new NullPointerException("You can only call getAudioTracks() when video is prepared.");
+        }
+
+        ArrayList<PlayerAudioTrack> audioTracks = new ArrayList<>();
+        MediaPlayer.TrackInfo[] trackInfos = mediaPlayer.getTrackInfo();
+
+        for (int i = 0; i < trackInfos.length; i++) {
+            MediaPlayer.TrackInfo trackInfo = trackInfos[i];
+            if (trackInfo.getTrackType() == MediaPlayer.TrackInfo.MEDIA_TRACK_TYPE_AUDIO) {
+                audioTracks.add(new PlayerAudioTrack(0, i, String.valueOf(trackInfo.hashCode()), trackInfo.getLanguage(), "", -1, -1));
```

Comment:
`i` is the absolute index because we iterate through all tracks and store only those audio ones.